app/src/main/res/values/strings.xml

```diff
@@ -472,4 +472,8 @@
     <string name="compose_shortcut_short_label">Compose</string>
     <string name="pref_title_bot_overlay">Show indicator for bots</string>
 
+    <string name="notification_clear_text">Are you sure you want to permanently clear all your notifications?</string>
+    <string name="yes">Yes</string>
+    <string name="no">No</string>
```

Comment:
there is `android.R.string.yes` and `android.R.string.no`, no need to define it ourselves